tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -1307,6 +1307,42 @@ public function test_couldnt_connect_notice_is_suppressed_during_outage() {
+
+		$notices = $this->getMockBuilder( WC_Stripe_Admin_Notices::class )
+			->setMethods( [ 'get_environment_type' ] )
+			->getMock();
+		$notices->method( 'get_environment_type' )->willReturn( $environment_type );
+
+		$notices->check_api_outage();
+
+		$this->assertSame( $expect_notice, isset( $notices->notices['api_outage'] ) );
+
+		WC_Stripe_API_Outage_Status::record_success();
+	}
```

PHPMD (2.15.0)

[error] The method test_api_outage_notice_is_gated_by_environment is not named in camelCase. (undefined)

(CamelCaseMethodName)